scripts/pre_commit_linter.py

```diff
@@ -468,6 +468,57 @@ def _pre_commit_linter(all_files):
     return summary_messages
 
 
+def _check_newline_character(all_files):
+    """This function is used to check new line characters.
+    """
+    print 'Starting to check new line at EOF'
```

Comment:
nit: Starting newline-at-EOF checks

(to follow the style of the other messages)